src/ar.erl

```diff
@@ -111,7 +113,7 @@ main("") ->
 		end,
 		[
 			{"peer (ip:port)", "Join a network on a peer (or set of peers)."},
-			{"start_hash_list (file)", "Start the node from a given block."},
+			{"start_hash_list (hash)", "Start the node from a given block."},
```

Comment:
This doc is not correct anymore. The hash is a hash of the block hash list. I think the argument name ("start_hash_list") is confusing as well, but ok.